docs/contributing/how-to-contribute-to-docs.md

```diff
@@ -79,6 +79,12 @@ Copy your company's logo in `website/static/logos`, and make sure it follows the
 * Greyscale is prefered but color is fine
 * Keep it under 50KB
 
+Also please make sure your company meets these requirements:
```

Comment:
Let's not repeat ourselves and be more professional. So intead of this section, can we just change line 68?

```
We'd love to showcase your company's logo on our main page and README!
Requirements for the company:
* it is using Thanos on production
* it is a legal registered company
* it is happy to announce that you use Thanos publically

If all those are met, add yourself in  ...   and create PR against Thanos `master` branch.

```

Reply:
What do you think? (:

Reply:
Yes, that looks more consistent. Thanks I reworekd it as you suggested hopefully if it's ok this way?